plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -89,6 +89,10 @@ public function handle( $timestamp, $level, $message, $context ) {
 
 			WC_Rate_Limiter::set_rate_limit( self::RATE_LIMIT_ID, self::RATE_LIMIT_DELAY );
 
+			if ( $this->is_dev_or_local_environment() ) {
+				return false;
```

Comment:
@chihsuan  That got me thinking, perhaps a good follow-up would be to put the log into WooCommerce status page or debug.log for non-production environments 

Or would it already be logged there since it's a fatal 🤔

Reply:
Yes, we always log PHP fatal errors into the WooCommerce status page.